### ISSUE TITLE
Spacing issue in XFX Entry

### ISSUE THREAD

**Issue by hashimks** [XamFormsExtended/Xfx.Controls#41]
Hello,

I have tried using around 6 XFX entries one after the other in a vertical manner in a form. I could see that the spacing between the entries are more. Is there any way to reduce it?

**Comment by ChaseFlorell** [comment]
The reason for this is that the control is making space for the Error Text. To fix this you have to disable Error Error Text. This feature was added in #33 

Take a look at `ErrorDisplay`

**Comment by Umar3x** [comment]
Hello,

I'm having the same prob. I got a behavior, and I set the error text of the entry depending on a Regex. For that it is working well, and I must say that this control is very useful, thanks ! 

Prob is even if I set Error Text to null, space remains. So I had in mind that I should also set Error display considering my behavior. 

In the handle text changed I made something like this : 

```
 private void HandleTextChanged(object sender, TextChangedEventArgs e)
 {
 IsValid = Regex.IsMatch(e.NewTextValue, emailRegex, RegexOptions.IgnoreCase, TimeSpan.FromMilliseconds(250));
 var entry = (XfxEntry) sender;
entry.ErrorDisplay = !IsValid ? ErrorDisplay.Underline : ErrorDisplay.None;
entry.ErrorText = !IsValid ? "Input mail is not valid" : "";
 }
```
But it does not to work, space for the error text remains. 
I tried to set in the init of the entry the error display to none by default, and see what will happen when setting it to underline in the behavior when I set an error text but does not work too still have the space.
